tests/FeaturesTest.php

```diff
@@ -37,6 +38,12 @@ public function testFeatures(string $file): void
 
     private function analyze(string $file): int
     {
+        $laravelMajorVersion = (int) explode('.', app()->version())[0];
+
+        if (Str::contains($file, 'Features/Laravel8') && $laravelMajorVersion < 8) {
```

Comment:
Please consider using `compare_version()` with < 8.0.0

Reply:
I agree with @szepeviktor

Reply:
Nice! Didn't know this PHP function :-D